python/src/gen_gherkin/behavior_generator.aac

```diff
@@ -0,0 +1,14 @@
+generator:
+    name: Gherkin Behavior Generator
+    description: A component diagram generation action for Gen-Gherkin plugin.
+    sources:
+        - name: Gherkin Behavior diagram
+          data_source: model
```

Comment:
Not sure I understand what you mean.

Reply:
You can drill down within the data source definition, so we could say `data_source: model.behavior` We would then be able to update the model content parsing that we do because it would just be looking for the behavior list to be passed in to generate and the template.